scene/main/scene_tree.cpp

```diff
@@ -1953,6 +1953,36 @@ bool SceneTree::is_using_font_oversampling() const {
 	return use_font_oversampling;
 }
 
+void SceneTree::get_argument_options(const StringName &p_function, int p_idx, List<String> *r_options) const {
+
+	if (p_function == "change_scene") {
+		DirAccess *dir_access = DirAccess::create(DirAccess::ACCESS_RESOURCES);
```

Comment:
You should be able to use `DirAccessRef`, and thus drop the `memdelete` below.

Reply:
What's the difference?

Reply:
None, but you can't create a memory leak by mistake with `DirAccessRef`, so that should be used preferentially throughout the codebase where possible.